src/models/task/index.js

```diff
@@ -15,6 +15,11 @@ const TaskModel = (repo) => {
         return [task, err];
     };
 
+    const getAllCompletedTasks = async (userID) => {
+        const [task, err] = await repo.getAllCompletedTasks(userID);
+        return [task, err];
```

Comment:
Suuuper nit but you can just do `return await repo.getAllCompletedTasks(userID);`

Reply:
I could do that, but for the sake of consistency perhaps I shouldn't? Because all of the others are split into two statements. Or I could just make all of them have single return statements?

![Screenshot from 2020-05-13 00-43-26](https://user-images.githubusercontent.com/57078680/81785164-cea17a00-94b2-11ea-968b-36cbafdac88f.png)

Reply:
Ok good point, we can leave as is